project/profiles/capacity4more/modules/c4m/og/c4m_og_menu/c4m_og_menu.install

```diff
@@ -1,24 +0,0 @@
-<?php
-
-/**
- * @file
- * Install - Un-installer & updates for the c4m_og_menu module.
- */
-
-/**
- * Implements hook_install().
- *
- * Make sure that we have the "home" menu link in the custom menu.
- */
-function c4m_og_menu_install() {
-  $t = get_t();
-
-  // Create a Home link in the OG Menu.
-  $item = array(
```

Comment:
Same remark here; the "HOME" item is in the menu, but I can't find the place where this is created.